🔧 Refactor manual-install Compose.yml: Simplify Environment Variables

flll opened the pull request:
- Removed explicit values for environment variables in `docker-compose.yml`.
- Utilized default values for better flexibility and maintainability.
- Updated network configuration to use the default bridge driver.

Note: Using `network: default` is sufficient within Docker Compose; there's no need to create a separate `nextcloud-network` for all hosts. 🚀

docjyJ reviewed (changes requested):
It is a good idea to use the default network!

For the variables, maybe you should rename them in the `.env` because there is a small error with the REDIS_HOST_PASWORD.

Zoey2936 reviewed (changes requested):
File is created via script: https://github.com/nextcloud/all-in-one/blob/main/manual-install/update-yaml.sh - you need to modify the script not the file

flll commented:
Thank you for your reviews, Zoey2936, docjyJ, and szaimen.

I've made additional changes to the `update-yaml.sh` script:
- Removed .services[].networks
- Added a special process to remove =%%

Additionally, I've corrected the REDIS_HOST_PASSWORD variable.

Thank you all for your valuable input and contributions.

docjyJ commented on manual-install/update-yaml.sh:
```diff
@@ -41,6 +42,7 @@ sed -i '/AIO_TOKEN/d' containers.yml
 sed -i '/AIO_URL/d' containers.yml
 sed -i '/DOCKER_SOCKET_PROXY_ENABLED/d' containers.yml
 sed -i '/ADDITIONAL_TRUSTED_PROXY/d' containers.yml
+sed -i 's/\(^[[:space:]]*- \([^=]*\)\)=%\2%/\1/' containers.yml
```
```suggestion
sed -ie 's/\( *- \(\w*\)\)=\${\2\}/\1/' containers.yml
```

I don't really understand the purpose of this line... If it's for environment variables, this regex seems better.

```diff
-      - TURN_SECRET=${TURN_SECRET}
+      - TURN_SECRET
-      - SIGNALING_SECRET=${SIGNALING_SECRET}
+      - SIGNALING_SECRET
-      - ONLYOFFICE_SECRET=${ONLYOFFICE_SECRET}
+      - ONLYOFFICE_SECRET
```

Test online: https://sed.js.org/

docjyJ reviewed (changes requested):
Otherwise LGTM

docjyJ reviewed (approved):
LGTM :heart:

szaimen commented:
Hi @flll thanks a lot for this contribution! I've just invited you to the repo so that collaboration is easier :)

szaimen commented:
The changes look good in general, but I fear there is another script that needs to be adjusted. See https://github.com/nextcloud/all-in-one/blob/main/nextcloud-aio-helm-chart/update-helm.sh

I've for easier testing pushed your changes to a local branch: https://github.com/nextcloud/all-in-one/pull/5459
After accepting my invite to the repo, you can then go to https://github.com/nextcloud/all-in-one/actions/workflows/update-helm.yml and click the Run workflow button and select the enh/5393/local-branch branch to test your changes. See 
![image](https://github.com/user-attachments/assets/81a11295-bcdf-4bc3-8b2a-f2b903d5e710)

flll commented:
Thank you very much.
I have edited update-helm.yml, pushed it to the enh/5393/local-branch, and ran a workflow against the local branch. I believe it was successful.
I apologize for the trouble I've caused you. Once again, thank you for your assistance.

szaimen commented:
Merged in https://github.com/nextcloud/all-in-one/pull/5459. Thanks a lot for tackling this and not giving up! 😊
